### ISSUE TITLE
Epic: Make sure .NET is continuously improved

### ISSUE THREAD

**Issue by terrajobst** [dotnet/core#5410]
### Summary

We'll use this epic to collect bottom-up improvements coming from the engineering team.

### User Stories

User Stories under this Epic:

- [ ] [Title of the linked User Story](hyperlink to the User Story issue).

**Comment by danmosemsft** [comment]
@carlossanlop yup, you can add a checkbox for it above then it will link up in https://dotnetepics.azurewebsites.net/

**Comment by marek-safar** [comment]
I think this issue should be moved to dotnet/runtime
